posthog/temporal/workflows/squash_person_overrides.py

```diff
@@ -384,10 +401,19 @@ async def squash_events_partition(inputs: QueryInputs) -> None:
 
     for partition_id in inputs.partition_ids:
         activity.logger.info("Executing squash query on partition %s", partition_id)
-        sync_execute(
-            query,
-            {"partition_id": partition_id, "team_ids": inputs.team_ids, "latest_created_at": inputs.latest_created_at},
-        )
+
+        parameters = {
+            "partition_id": partition_id,
+            "team_ids": inputs.team_ids,
+            "latest_merged_at": inputs.latest_merged_at,
+        }
+
+        if inputs.dry_run is True:
+            activity.logger.info("This is a DRY RUN so nothing will be squashed.")
+            activity.logger.info("Would have run query: %s with parameters %s", query, parameters)
+            continue
+
+        sync_execute(query, parameters)
```

Comment:
nit: maybe we should always print out the cmds
```suggestion
        
        activity.logger.info(f"{inputs.dry_run ? 'Would have ran query:' : 'Running query:'} {query} with parameters {parameters}")
        if not inputs.dry_run:
            sync_execute(query, parameters)
```

Reply:
If the query actually runs there's the query log